src/BalanceTx.purs

```diff
@@ -227,25 +218,28 @@ newtype Expected = Expected Value
 derive instance newtypeExpected :: Newtype Expected _
 
 instance showExpected :: Show Expected where
-  show = show <<< flattenValue <<< unwrap
+  show = show <<< unwrap
```

Comment:
Let's make Show instances more copy-paste-able. I.e. not omit constructor name entirely. Also, it can be expressed as `derive newtype instance Show Expected`. But in this case I'd go with genericShow.

Reply:
I would also suggest `genericShow` here (and in general unless there is a very compelling reason not to use it)